src/styles/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+  padding-inline: 16px;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin-top: 32px;
+    margin-inline: 3px;
```

Comment:
The image should be centered using logical properties. Instead of setting margin-inline, consider using `margin: auto` to center the image horizontally.